src/api/index.spec.tsx

```diff
@@ -145,6 +145,36 @@ describe(__filename, () => {
     });
   });
 
+  describe('getVersionFile', () => {
+    it('calls the API to retrieve default version file information', async () => {
+      const versionId = 123;
+
+      await getVersionFile({ apiState: defaultApiState, versionId });
+
+      expect(fetch).toHaveBeenCalledWith(
+        `/api/v4/reviewers/browse/${versionId}/`,
+        {
+          headers: {},
+          method: HttpMethod.GET,
+        },
+      );
+    });
+    it('calls the API to retrieve information for a specific version file', async () => {
+      const filename = 'test.js';
+      const versionId = 123;
+
+      await getVersionFile({ apiState: defaultApiState, filename, versionId });
+
+      expect(fetch).toHaveBeenCalledWith(
+        `/api/v4/reviewers/browse/${versionId}/?file=${filename}`,
+        {
+          headers: {},
+          method: HttpMethod.GET,
+        },
```

Comment:
You could omit this since it is covered by another test case, but it is also fine if you want to highlight the fact that passing a file does not change the `headers`.